alt-fuel-finder/spec/models/station_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe Station, type: :model do
+  context "#nearest_stations_by_zip" do
+    it "should return station objects" do
+      zip_code = 80203
+      stations = Station.nearest_stations_by_zip(zip_code)
+      station = stations.first
+  
+      expect(station).to be_kind_of(Struct::Station)
```

Comment:
The object types, after going through Struct, were `Struct::Station`. I was initially testing `to_be_kind_of(Station)`, and sorta wish I could still be doing that.

If I were DDD'ing this, I'd expect `Station`, not `Struct::Station`.

Is it appropriate to expect this current version, or is there a way I can "trick" Rails into thinking these are all Station objects?